.stylelintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: [
+    'stylelint-config-standard',
+    'stylelint-config-prettier',
+    'stylelint-config-recommended',
+    'stylelint-config-styled-components',
```

Comment:
Do we need this? According to docs, this is needed when you're using `stylelint-processor-styled-components`. With `stylelint` supporting this out of the box it doesn't seem needed

Reply:
yeah, we don't need that, we don't need also `config-prettier'.
Had to ignore one rule more which is